packages/grafana-data/src/datetime/formats.test.ts

```diff
@@ -0,0 +1,20 @@
+import { localTimeFormat } from './formats';
+
+describe('Date Formats', () => {
+  it('localTimeFormat', () => {
+    const format = localTimeFormat(
+      'default',
+      {
+        year: '2-digit',
+        month: '2-digit',
+        day: '2-digit',
+
+        hour: '2-digit',
+        minute: '2-digit',
+        second: '2-digit',
+      },
+      ''
+    );
+    expect(format).toBe('MM/DD/YYYY, HH:mm:ss A');
```

Comment:
Hi!

`YYYY` comes from this mapping: https://github.com/alex-shpak/grafana/blob/feature/graph-local-dates/packages/grafana-data/src/datetime/formats.ts#L20
It transform any year field to momentjs format `YYYY`

Something to keep in mind: Tests are running in Node environment, not a real browser. So it is using node's Intl api for this. So that comma comes from `Node` formatting.

In node 12
```
new Intl.DateTimeFormat('default', options).formatToParts(new Date());
[
  { type: 'month', value: '08' },
  { type: 'literal', value: '/' },
  { type: 'day', value: '25' },
  { type: 'literal', value: '/' },
  { type: 'year', value: '20' },
  { type: 'literal', value: ', ' },
  { type: 'hour', value: '05' },
  { type: 'literal', value: ':' },
  { type: 'minute', value: '00' },
  { type: 'literal', value: ':' },
  { type: 'second', value: '16' },
  { type: 'literal', value: ' ' },
  { type: 'dayPeriod', value: 'PM' }
]
```

see `{ type: 'literal', value: ', ' },` after `{ type: 'year', value: '20' },`